app/views/docs/getting-started-for-flutter.phtml

```diff
@@ -121,7 +121,7 @@ close the window.
 <p>You can also install the SDK using the <a href="https://pub.dev/" target="_blank" rel="noopener">Dart package manager</a> from your terminal:</p>
 
 <div class="ide margin-bottom" data-lang="bash" data-lang-label="Bash">
-  <pre class="line-numbers"><code class="prism language-bash" data-prism>pub get appwrite</code></pre>
+  <pre class="line-numbers"><code class="prism language-bash" data-prism>pub add appwrite</code></pre>
```

Comment:
This won't actually run. The correct command should be:

```suggestion
  <pre class="line-numbers"><code class="prism language-bash" data-prism>flutter pub add appwrite</code></pre>
```

Reply:
Fixed